bakalari_homework_tasklist/core.py

```diff
@@ -17,11 +17,10 @@
 
 bp = Blueprint("core", __name__)
 
-
```

Comment:
Please format the code with the `black` formatter or at least re-add these newlines. Sorry there's no contributing guide about code style, I will add it soon

Reply:
ok. Fixed in 24f755e3ad6abe5cdcc80330a8c7ccbd578117d6